### PR TITLE
Fix issue in ethnicity and another address question

### LINKED ISSUE #1355: Issue in Ethnicity and Another Address Questions
### Expected behaviour
Guidance above question and not to include "please specify" in label.

### Actual behaviour

Ethnicity guidance located in the wrong place
Do you stay at another Address still has the "please specify" label on an answer checkbox.

### Steps to reproduce the behaviour

Run Census Household and navigate to relevant questions to view 

### Technical information

Easy fix

#### Browser

All

#### Operating System

All

### Screenshot

CBA

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?

- moved guidance to above question for ethnicity question

- removed please specify from label in another address question

Card: https://trello.com/c/tEyntfnt/1511-issue-in-ethnicity-and-another-address-questions

fixes: #1355

### How to review 
The usual check run Census Household and verify changes made.

### CODE REVIEW

**Review by shahi645 — APPROVED** [review]
LGTM

File: data/en/census_household.json [hunk 1]
```diff
@@ -1952,24 +1966,22 @@
                         {
                             "id": "ethnic-group-wales-question",
                             "title": "What is your ethnic group?",
+                            "guidance": {
+                                "content": [{
+                                        "description": "It is for you to choose your ethnic group. For example this could be your family or cultural background, or it could be how you would describe yourself."
+                                    },
+                                    {
+                                        "description": "Please select the answer you think is most appropriate."
```
Comment (charlotte-hirst) [inline data/en/census_household.json]: this line should not be there as per card

File: data/en/census_household.json [hunk 2]
```diff
@@ -1952,24 +1966,22 @@
...
+                                    },
+                                    {
+                                        "description": "If you don’t feel the options apply to you, select ‘Other ethnic group’. You will then be able to write in the ethnic group you prefer to identify as at the next question."
```
Comment (charlotte-hirst) [inline data/en/census_household.json]: this line should not be there as per card

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]
guidance shouldnt be in a panel unless its an include or exclude

**Comment by MIslam92** [comment]
@charlotte-hirst can you have a approve this so I can get this merged?

**Review by charlotte-hirst — APPROVED** [review]

**Pull request merged**